src/couch_replicator/include/couch_replicator_api_wrap.hrl

```diff
@@ -26,13 +26,6 @@
     httpc_pool = nil,
     http_connections,
     first_error_timestamp = nil,
-    proxy_url
-}).
-
--record(oauth, {
-    consumer_key,
-    token,
-    token_secret,
-    consumer_secret,
-    signature_method
+    proxy_url,
+    auth_context = nil
```

Comment:
before I go deeper, we're changing the default for one field and adding an extra property. Are we sure there's no upgrade problem here? I think we never pass httpdb's between nodes but this would be bad if we did.

Reply:
I was of that. We never pass #httpd{} record between nodes